### ISSUE TITLE
porting to sdk 17

### ISSUE THREAD

**Issue by NorwegianSailor** [jimmywong2003/SSD1306-LCD-on-NRF52#2]
Dear sir,
I am using your ported driver in a ‘hobby’ project where I use Nordic SDK 17.0.2.
If I remove everything related to BLE in your port of Adafruit SSD1306 driver it compiles and run on a PCA10056 DK without any problem.
With other words, everything related to TWI/TWIM works fine in SDK17.0.2
So, to my problem, I want to merge the stripped version to my current project based on Nordic NUS service (UART over BLE). 
If I do so I get the following error message when building:
'NRFX_TWIM0_INST_IDX' undeclared here (not in a function); did you mean 'NRFX_TWI1_INST_IDX'?

I have googled and read through lot of discussion related to this fault on the nett as well as Nordic developer forum and I have tried all the proposal given there without any success. I have tried to understand the SDK-CONFIG file. Your SDK_CONFIG file is around 3600 lines, but the one used for NUS example is more than 14000 lines so I believe the problem might be related to SDK_CONFIG, but not sure.
If you have an idea of what could be the problem, please point me into that direction and I will try to figure out what is going on.

Kind regards
Svein, retired and have not made real C code since early 90s.

**Comment by NorwegianSailor** [comment]
Dear sir,
I forgot to mention that I used the Nordic SDK17.0.2  example  'twi_sensor' as a base when I merged /ported yorur  stripped (without BLE)  code to sdk17.0.2. 
Kind regards
Svein

**Comment by NorwegianSailor** [comment]
Dear sir,
I am sorry to have bothered you.
It looks like that I have in one way or the other mixed the SSD1306.c and SSD1306_nrfx_twim.c in the Adfruit driver folder when I merged them into my application in SDK17.0.2. 
When I removed SSD1306_nrfx_twim.c and added SSD1306.c into my project everything worked.
Now I have to figure out how to disable TWI when I have powered off the display and moved into System on Idle in order to save power. I want less than 4 micro ampere when in System ON Idle.
I apologize for the inconvenience I have caused and apologize.
